client/src/locales/en-US.json

```diff
@@ -503,12 +503,14 @@
             "dropInstructions": "Drag & drop your files to import them"
         },
         "CopyFile": {
+            "copy": "Copy of",
```

Comment:
It would be better to use formatting:
Have `Copy of {name}` in your translation files, and then use `$msTranslate({ key: 'key', data: { name: shortenFileName(...) }})`